packages/components/src/TreeView/TreeView.component.js

```diff
@@ -92,7 +92,7 @@ function TreeView({
 				<ul className={theme['tc-treeview-ul']}>
 					{structure.map((item, i) => (
 						<TreeViewItem
-							id={id && `${id}-${i}`}
+							id={`${id}-${i}`}
```

Comment:
this change is not listed on the pr description, and potentially break QA tests, it need to be listed as Breaking change

Reply:
id has been added as not required to help QA.
I prefer add a default value so I don't need to check
Description added to the PR

Reply:
We should align how ids work within our components. If no id in props, then no generated ids, same for add (all) button